### PR TITLE
add AaquibAli

### LINKED ISSUE #747: AaquibAli
aaquib

### PR DESCRIPTION
**Pull request by AaquibAli**
resolve #747

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1385,4 +1385,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/73016212/135509959-87f8b8e4-9fab-4247-bae4-d965fca5ddbc.png",
     "issueNumber": 742
   }
```
Comment (schmelto) [inline data.json:1387]:
```suggestion
  },
```

File: data.json [hunk 2]
```diff
@@ -1385,4 +1385,10 @@
...
+  {
+    "name": "Aaquib Ali",
+    "githubUsername": "AaquibAli",
+    "imageUrl": "![aaquib](https://user-images.githubusercontent.com/84700316/135708541-936038fd-97c2-4fc0-a51d-7d6a5b96dbf1.png",
```
Comment (schmelto) [inline data.json:1391]:
```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/84700316/135708541-936038fd-97c2-4fc0-a51d-7d6a5b96dbf1.png",
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by AaquibAli** [comment]
i change the review. please have a look @schmelto

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
**Thanks for your contribution** 🚀 

I hope you enjoyed the process and we see us in some other PRs again.

**Comment by AaquibAli** [comment]
Thanks for merging this PR

**Comment by takasurazeem** [comment]
Profile no longer exists

**Pull request merged**